rpc/src/rpc_subscription_tracker.rs

```diff
@@ -575,6 +575,7 @@ impl Drop for SubscriptionTokenInner {
     }
 }
 
+#[allow(dead_code)]
```

Comment:
This one isn't as immediately obvious to me, fine with giving it `dead code` for now too

Reply:
Good point, I agree that a one-line comment would be good. Save someone who is looking at it the trouble of having to remove the `#[allow(dead_code)]` to see the compiler warning

Reply:
okay!!! just added it at [a7e6901](https://github.com/anza-xyz/agave/pull/1246/commits/a7e690117b08a45823b55493c0e393a64ac27307) 🫡